db/Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM postgis/postgis:15-3.4-alpine
+FROM postgres:15-bookworm
```

Comment:
Am I correct in assuming docker and docker compose are smart enough to just figure out which architecture they should use based on the user's machine? I set a `platform` param in `compose.yaml` when I was playing around with this but not sure if it was actually necessary.

Reply:
From the documentation, [docker says](https://docs.docker.com/engine/reference/builder/)

> By default, the target platform of the build request is used


... I have no idea what that means. But, I did not set the `platform` a param and had no issues. My sanity check was looking for a warning in the docker desktop container page. The warning says something like "container may be slow or fail because it's not meant for this platform". I had the warning for the previous image and did not get it here.

Reply:
Sorry, could have been clearer. I'll rephrase the question: Does this set up use the x86 version of the image on x86 machines and the arm version on arm machines? Sounds like it does.